sw/device/silicon_creator/manuf/data/manuf_testplan.hjson

```diff
@@ -0,0 +1,408 @@
+            - Verify lc state.
+
+            Implement the following steps in a way in which it could be executed at a later time
+            in silicon targets:
+
+            - Load OTP state generted by previous steps.
+            - Select LC TAP via strap pins.
+            - Request transition to TEST_UNLCOKED by providing the TEST_UNLOCK_TOKEN provisioned
+              in previous steps.
+            - Perform reset by toggling POR pin.
+            - Select LC TAP via strap pins.
+            - Verify lc_state.
+
+            Repeat the same flow for the number of supported TEST_LOCK/UNLOCKED states.
+            '''
+      tags: ["dv", "fpga", "silicon"]
+      stage: V3
+      tests: []
+    }
+
+    {
+      name: manuf_ft_exit_token
+      desc: '''Verify provisioning and use of test exit token.
+
+            The test exit token is used to move the device from test mode into DEV, PROD or
+            PROD_END state. This test also incorporates enabling of ROM execution before
+            triggering the lc_state change.
+
+            - Pre-load OTP with TEST_UNLOCKED lc_state in pre-silicon environments. Advance to
+              state in silicon.
+            - Select CPU JTAG via strap pins.
+            - Program TEST_EXIT_TOKEN in OTP transition via JTAG.
+            - Reset device by toggling POR pin.
+            - Select CPU JTAG via strap pins.
+            - Program CREATOR_SW_CFG_ROM_EXEC_EN in OTP via JTAG.
+            - Select LC TAP via strap pins (at runtime).
+            - Request transition into {pick one: DEV, PROD, PROD_END} state providing TEST_EXIT_TOKEN.
+            - Perform reset by toggling POR pin.
+            - Select LC TAP via strap pins.
+            - Verify lc state.
+
+            After this test most of the following manuf_ft_* test cases can be loaded into flash via
+            boostrap in silicon.
+            '''
+      tags: ["dv", "fpga", "silicon"]
+      stage: V3
+      tests: []
+    }
+
+    {
+      name: manuf_ft_sku_individualization
+      desc: '''Configure device fuses.
+
+            This includes configuration for most countermeasures and hardware options. It does not
+            include configuration of the device identity and RMA token, which is done in the
+            personalization step.
+
+            The test steps are expected to be executed from flash. In silicon, the program is loaded
+            using the ROM bootstrap protocol. It is recommended to use the same approach on the FPGA,
+            and enable it as a configuration option in DV.
+
+            - Pre-load OTP with {pick one: DEV, PROD, PROD_END} lc_state in pre-silicon
+              environments. Advance to state in silicon.
+            - Load test program into flash and start execution.
+            - Check device lc_state. Return error to the host if device in invalid state.
+            - Configure the following OTP fuses to enable access to entropy:
```

Comment:
@alphan We need to determine how to enabling flash scrambling as the Silicon Creator is going to skip scrambling for the data partition. One option is to only enable scrambling for the ROM_EXT partition.

Reply:
As we discussed this in the meeting, these settings must be performed before bootstrap in ROM. For ROM_EXT, we actually need two entries because of the dual-slot, i.e. a/b, layout. we'll need the same for the owner region as well. WDYT about exposing all 8 entries in the OTP? I think we can exclude read, write, and enable so that they don't interfere with the bootstrap process (all enabled by default). The owner can set base, size, execute, scrambling, ECC, and HE (can leave out if we can run out of space), which should fit in a 32-bit word.

For info pages, I think we only need to worry about the pages that are accessed by ROM since all others can be handled in ROM_EXT or the owner stage. We already handle boot_data pages in `flash_ctrl_init()`.